core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -72,7 +72,7 @@ class ReplicaFetcherThread(name: String,
       metrics,
       time,
       "replica-fetcher",
-      Map("broker-id" -> sourceBroker.id.toString).asJava,
+      Map("thread-id" -> name).asJava,
```

Comment:
thread-id is not a very meaning tag since name has both the broker id and the thread-id. It would be better to pass in the thread id and add a tag for broker-id as well as a thread-id.